Include/cpython/code.h

```diff
@@ -90,7 +90,7 @@ typedef uint16_t _Py_CODEUNIT;
     PyObject *co_qualname;        /* unicode (qualname, for reference) */      \
     PyObject *co_linetable;       /* bytes object that holds location info */  \
     PyObject *co_weakreflist;     /* to support weakrefs to code objects */    \
-    void *_co_code;               /* cached co_code object/attribute */        \
+    PyObject *_co_code;               /* cached co_code object/attribute */    \
```

Comment:
```suggestion
    PyObject *_co_code;           /* cached co_code object/attribute */        \
```
Align with other comment